### DISCUSSION TITLE
video call without peer

### DISCUSSION THREAD

**Discussion by veeyka** [peers/peerjs#1081] [Q&A]
hi . is there a way to implement video call without peer . my server is node js and ui is react js . I have done this with peer js before . but I want to know a way without peer js or a 3d party api . please give me a document tutorial if there is any
thanks

**Comment by samhirtarif** [comment]
PeerJS is a wrapper over the [WebRTC](https://developer.mozilla.org/en-US/docs/Web/API/WebRTC_API) framework. And this wrapper abstracts out a lot of complexities that you would otherwise face if implementing it yourself. However, it is doable, and you could follow the [documentation on MDN](https://developer.mozilla.org/en-US/docs/Web/API/WebRTC_API/Signaling_and_video_calling) for that.
